docs/how-tos.md

```diff
@@ -1,6 +1,7 @@
 <!-- MarkdownTOC autolink=true -->
 
 - [How-to integrate with an existing store ?](#how-to-integrate-with-an-existing-store-)
+- [How to use the client on Node environment ?](#use-the-client-on-node-environment-referenceerror-fetch-is-not-defined)
```

Comment:
The anchor is not correct since it has not the how-to

Reply:
Sorry for those typos :/